playbooks/nginxplus_staging.yml

```diff
@@ -0,0 +1,66 @@
+---
+# you MUST run this playbook on a single host with '--limit' for example `ansible-playbook -v --limit lib-adc2.princeton.edu playbooks/nginxplus.yml`
+# to update the second load-balancer, switch which adc# host is currently active, then run the playbook on the second (formerly active, now inactive) host
+# to update configuration for existing sites, run with '-t update_conf' for example `ansible-playbook -v --limit lib-adc2.princeton.edu playbooks/nginxplus.yml -t update_conf`
+# if you want to remove obsolete config files, you must pass '-e nginx_cleanup_config=true'
+# note that this will delete the entire config directory
+# To find the active load-balancer ssh onto a machine and run `ip a |grep 146` the active one will display `inet 128.112.203.146/32 scope global eno1`
+
+- name: update loadbalancer configuration
```

Comment:
I'm a little confused about why there are two playbook files if there's a staging and prod environment, and why they both point to the same hosts.

Reply:
Until this is ready to go in, this is a safety precaution. While this is tested fairly well, there are still moving parts on our dev environment that I am not quite ready to apply. I can be convinced otherwise but it feels like a useful guard as we transition.

Reply:
Is your concern that we might run the `staging` playbook against production LBs and get unexpected changes in production?